CHANGELOG.md

```diff
@@ -1,6 +1,38 @@
 Change Log
 ==========
 
+Version 3.0.0-rc02
+---------------------
+
+_2019-10-01_
+
+### Kotlin
+
+ * Fix: Nullify other oneof fields in Builder setters in Kotlin interop.
+ * Fix: Use unknownFields in `hashCode()`.
+ * Fix: Remove `withoutUnknownFields()` from Kotlin.
+
+### gRPC
+
+ * Update: Total rewrite of the generated interfaces for clients:
+
+ Introduce two interfaces, `GrpcCall` for simple RPCs, and `GrpcStreamingCall` fox duplex ones. Both
+ will provide blocking and suspending API, including a reference to the underlying
+ [OkHttp](https://github.com/square/okhttp)
+ [Call](https://square.github.io/okhttp/4.x/okhttp/okhttp3/-call/) object and its timeout.
```

Comment:
![Screen Shot 2019-10-01 at 1 55 33 PM](https://user-images.githubusercontent.com/133019/65987128-3a1e6600-e453-11e9-937f-839d2de05ebe.png)

Reply:
I think it’s not indented enough?